### PR TITLE
feat: Added unit test templates

### PR DESCRIPTION
**Pull request by KreativJos**
Templates for unit tests
#71 

XUnit
NUnit
MSTest

### CODE REVIEW

File: templates/mstest.tmpl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+${namespaces}namespace ${namespace}
```
Comment (bard83) [inline templates/mstest.tmpl:1]: I assume this will be the injected  somehow based on the test location, right?

**Comment by bard83** [comment]
Nice to see this PR. Thank you. I see that it was also easy to add the new menu items.

Well done.

File: templates/mstest.tmpl (hunk 1 above)
Comment (KreativJos) [reply templates/mstest.tmpl:1]: Do you mean the `using XUnit` etc (${namespaces} template var}? That is covered by the array part in these lines: https://github.com/KreativJos/csharpextensions/blob/77eeffed180ba780ca85a7e23187f5d35008781a/src/extension.ts#L155-L157

The ${namespace} is handled like all other templates, via `NamespaceDetector.getNamespace` - https://github.com/KreativJos/csharpextensions/blob/77eeffed180ba780ca85a7e23187f5d35008781a/src/namespaceDetector.ts#L16

**Comment by bard83** [comment]
The PR has conflicts that need to be addressed before it can be merged.

**Pull request merged**